storage/storage.go

```diff
@@ -241,6 +245,26 @@ type RefreshToken struct {
 	Nonce string
 }
 
+// RefreshTokenRef is a reference object that contains metadata about refresh tokens.
+type RefreshTokenRef struct {
+	ID        string
+	CreatedAt time.Time
+	LastUsed  time.Time
+}
+
+// OfflineSession objects are sessions pertaining to users with refresh tokens.
+type OfflineSession struct {
+	// UserID of an end user who has logged in to the server.
+	UserID string
+
+	// The ID of the connector used to login the user.
+	ConnID string
+
+	// RefreshTokenRef object which is a reference object
+	// for refresh tokens issued to a user with ID = UserID.
+	Refresh RefreshTokenRef
```

Comment:
Sorry I missed this, but a user can have multiple RefreshTokens out for different clients.